aspnetcore/signalr/javascript-client.md

```diff
@@ -66,6 +66,9 @@ The following code creates and starts a connection. The hub's name is case insen
 
 Typically, browsers load connections from the same domain as the requested page. However, there are occasions when a connection to another domain is required.
 
+> [!NOTE]
+> The client code will need to use a full url now instead of a partial url. Change `.withUrl("/chathub")` to `.withUrl("https:/myappurl/chathub")`.
```

Comment:
```suggestion
> [!IMPORTANT]
> The client code must use an absolute URL instead of a relative URL. Change `.withUrl("/chathub")` to `.withUrl("https://myappurl/chathub")`.
```